int16 requested_state
---
string msg_feedback
